### PR TITLE
Adding style to sign in form

### PR DESCRIPTION
**Pull request by Anapaotrev**
### What does this PR do?

* Styling the sign in form with bootstrap !

bitmoji

### CODE REVIEW

File: app/assets/stylesheets/signin_form.scss [hunk 1]
```diff
@@ -0,0 +1,55 @@
+@import "bootstrap";
```
Comment (kurenn) [inline app/assets/stylesheets/signin_form.scss]: Why are you importing bootstrap in here? isn't already on the `application.scss` file?

File: app/assets/stylesheets/signin_form.scss [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+
+body {
+  background-color: #fff;
+  padding-top: 30px;
+}
+
+.border-form-div {
```
Comment (kurenn) [inline app/assets/stylesheets/signin_form.scss]: Please order the properties for the css alphabetically

Comment (kurenn) [inline app/assets/stylesheets/signin_form.scss]: I don't feel quite comfortable with this class name, it contain html element names on the class name...and also a property, so I don't know what it is suppose to do...

File: app/views/devise/sessions/new.html.erb [hunk 3]
```diff
@@ -1,26 +1,34 @@
-<h2>Log in</h2>
+<%= stylesheet_link_tag "signin_form" %>
```
Comment (kurenn) [inline app/views/devise/sessions/new.html.erb]: This style guide will be included in the middle of the HTML, that is not quite right, check out the `content_for` method for Rails to include this on the header dinamically

File: app/views/devise/sessions/new.html.erb [hunk 4]
```diff
@@ -1,26 +1,34 @@
...
 
-<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :email %><br />
-    <%= f.email_field :email, autofocus: true %>
-  </div>
+<div class= 'border-form-div'>
```
Comment (kurenn) [inline app/views/devise/sessions/new.html.erb]: On html you don't separate the `=` sign from the value

**Review by kurenn — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/signin_form.scss (hunk 1 above)
Comment (Anapaotrev) [reply app/assets/stylesheets/signin_form.scss]: It didn't work we didn't know why, and when we changed it and then it worked.

File: app/assets/stylesheets/application.scss [hunk 5]
```diff
@@ -12,4 +12,16 @@
  *
  */
 
-@import "bootstrap";
\ No newline at end of file
+@import "bootstrap";
+
+.btn-purple {
```
Comment (kurenn) [inline app/assets/stylesheets/application.scss]: I would not set the class name to attached to a css property value, what would happen if tomorrow you decide purple is not the right color and change to, let's say orange, you will have to update the class name in order for the button to make sense at all....

File: app/views/devise/sessions/new.html.erb (hunk 3 above)
Comment (kurenn) [reply app/views/devise/sessions/new.html.erb]: Are we checking this?

File: app/assets/stylesheets/signin_form.scss (hunk 1 above)
Comment (kurenn) [reply app/assets/stylesheets/signin_form.scss]: So?

**Review by kurenn — CHANGES_REQUESTED** [review]

**Comment by kurenn** [comment]
norris

File: app/assets/stylesheets/signin_form.scss [hunk 6]
```diff
@@ -0,0 +1,31 @@
+a {
+  color: #A486B6;
+}
+
+a:hover {
+  color: #532E69;
+}
+
+body {
+  background-color: #fff;
```
Comment (kurenn) [inline app/assets/stylesheets/signin_form.scss]: Please update this to be upcased, I can see you are using upcased color values...

File: app/assets/stylesheets/signin_form.scss [hunk 7]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+.container {
+  -webkit-border-radius: 5px;
+     -moz-border-radius: 5px;
+          border-radius: 5px;
+  -webkit-box-shadow: 0 1px 2px rgba(0,0,0,.05);
+     -moz-box-shadow: 0 1px 2px rgba(0,0,0,.05);
+          box-shadow: 0 1px 2px rgba(0,0,0,.05);
+  background-color: #fff;
```
Comment (kurenn) [inline app/assets/stylesheets/signin_form.scss]: Please update this to be upcased, I can see you are using upcased color values...

File: app/assets/stylesheets/signin_form.scss [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+  border: 1px solid #e5e5e5 ;
```
Comment (kurenn) [inline app/assets/stylesheets/signin_form.scss]: Please update this to be upcased, I can see you are using upcased color values...

**Review by kurenn — CHANGES_REQUESTED** [review]

File: app/assets/stylesheets/signin_form.scss [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+
+.container {
+  -webkit-border-radius: 5px;
+     -moz-border-radius: 5px;
+          border-radius: 5px;
+  -webkit-box-shadow: 0 1px 2px rgba(0,0,0,.05);
+     -moz-box-shadow: 0 1px 2px rgba(0,0,0,.05);
+          box-shadow: 0 1px 2px rgba(0,0,0,.05);
+  background-color: #FFF;
+  border: 1px solid #E5E5E5 ;
```
Comment (kurenn) [inline app/assets/stylesheets/signin_form.scss]: Remove the space between the color and the `;`

**Review by kurenn — CHANGES_REQUESTED** [review]

**Review by kurenn — APPROVED** [review]

**Pull request merged**